price_pusher/tsconfig.json

```diff
@@ -1,13 +1,15 @@
 {
+  "extends": "../tsconfig.base.json",
   "compilerOptions": {
     "target": "esnext",
     "module": "commonjs",
     "declaration": true,
+    "rootDir": "src/",
     "outDir": "./lib",
     "strict": true,
     "esModuleInterop": true,
     "resolveJsonModule": true
   },
   "include": ["src"],
-  "exclude": ["node_modules"]
+  "exclude": ["node_modules", "**/__tests__/*"]
```

Comment:
Does price pusher have tests?

Reply:
No but I want to keep the tsconfigs all consistent